src/plugin/dataPopulator.ts

```diff
@@ -50,6 +52,19 @@ async function getPull(variable) {
   return await getDataFromAPI(route);
 }
 
+async function getApp(variable) {
+  const previewHeaders = {
+    headers: {
+      Accept: 'application/vnd.github.machine-man-preview+json',
```

Comment:
@matthewferry if this is what we need to make it work, would you mind adding a comment or something here that points to the documentation or the reason for needing this? Thank you!

Reply:
Absolutely. ✅